vdb-bench-assembly/plugins/vdb-bench-dataservice/export/dsExportGitWizard.js

```diff
@@ -90,7 +103,25 @@
                 return false;
             }
 
-            return true;
+            if ( $rootScope.requireAuthorName && _.isEmpty(vm.repo.parameters['author-name-property'])) {
+                return false;
+            }
+
+            if ( $rootScope.requireAuthorEmail && _.isEmpty(vm.repo.parameters['author-email-property'])) {
+                return false;
+            }
+
+            // see if HTTP authentication is complete
+            if ( !_.isEmpty( vm.repo.parameters[ 'repo-password-property' ] ) && !_.isEmpty( vm.repo.parameters[ 'repo-username-property' ] ) ) {
+              return true;
+            }
+            
+            // see if SSH authentication is complete
+            if ( !_.isEmpty( vm.repo.parameters[ 'repo-password-property' ] ) && !_.isEmpty( vm.repo.parameters[ 'repo-known-hosts-property' ] ) ) {
+              return true;
```

Comment:
These validation tests do not make sense. A git repository can use http authentication, ssh authentication and even have no authentication at all. As such, I think it difficult to try to assume which of these properties should be populated. For example, it is 'normal' for the known-hosts to be required for ssh but if the ssh server is configured to not use known-hosts then this is unnecessary. Thus, my recommendation would be to remove these tests at this time.